devtools/readme.md

```diff
@@ -55,9 +56,9 @@ So we've seen that fonts, colors, and spacing are often stored as CSS Variables,
 4. Try changing the values of these variables and see how it affects the page.
 5. Some questions to answer. (Make a prediction about what you think will happen first, and then test it)
 
-- What will happen when you change the _value_ of `--space` to `red`?
 - What will happen when you click the value of `--finger` and press `up arrow` ten times?
 - What will happen when you delete the _value_ of `--copy`
+- What will happen when you change the _value_ of `--space` to `red`?
```

Comment:
I'm moving this to the end of the list because I think we should establish "how are things meant to work" before introducing "what happens if you do something wrong" :)